src/JSmodules/Home.js

```diff
@@ -0,0 +1,23 @@
+export class Home {
+    constructor() {
+        this.textElement = document.querySelector('.view__textContent');
+    
+        this.start();
+    };
+
+    start() {
+        setInterval(() => {
+            this.fullDate = new Date();
```

Comment:
I will recomend to put this interval in separate method e.g. _timer() or something like this